packages/expo/cli/utils/user/otp.ts

```diff
@@ -0,0 +1,179 @@
+import assert from 'assert';
```

Comment:
@wschurman Could you look over the login/auth-related files? I believe they are taken straight from the other CLIs for the most part so just need a sanity check

Reply:
LGTM